lib/env.js

```diff
@@ -0,0 +1,9 @@
+const SAO_GLOBAL_SCOPE = process.env.SAO_GLOBAL_SCOPE || 'sao'
+const SAO_MODULE_PREFIX = process.env.SAO_MODULE_PREFIX || 'sao'
+const SAO_CONFIG_FILE = process.env.SAO_CONFIG_FILE || 'saofile'
+
+module.exports = {
+  SAO_GLOBAL_SCOPE,
+  SAO_MODULE_PREFIX,
+  SAO_CONFIG_FILE
+}
```

Comment:
`cacheDir` `modulePrefix` `validConfigFiles`

Reply:
But when I started to work for these API options, I find that we need to import many unnecessary code changes.

e.g. `store` is more like a global state center, it will be introduced in multiple places. and we'll need to initialize it explicitly every time.